src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -238,7 +238,7 @@ int sd_dhcp_client_set_callback(
 int sd_dhcp_client_set_request_broadcast(sd_dhcp_client *client, int broadcast) {
         assert_return(client, -EINVAL);
 
-        client->request_broadcast = !!broadcast;
+        client->request_broadcast = broadcast;
```

Comment:
Are we sure about this one? I see it often used to convert other types to bool

Reply:
The rationalization comes from 5d904a6aaaceae7fe2f11d6b848b0dd45e3fd1c4 so I guess it's the way to go?

Reply:
thanks, makes sense